Saves having to rebuild valgrind on glibc-x.x.* patch level releases

diff -Naur valgrind-3.5.0-old//glibc-2.X.supp.in valgrind-3.5.0/glibc-2.X.supp.in
--- valgrind-3.5.0-old//glibc-2.X.supp.in	2009-08-19 23:37:48.000000000 +1000
+++ valgrind-3.5.0/glibc-2.X.supp.in	2010-05-22 19:43:08.088007038 +1000
@@ -168,9 +168,9 @@
    Memcheck:Param
    socketcall.sendto(msg)
    fun:__sendto_nocancel
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
 }
 {
    glibc24-64bit-padding-1c
@@ -180,7 +180,7 @@
    fun:__nscd_get_map_ref
    fun:nscd_get*_r
    fun:*nscd*
-   obj:/*libc-@GLIBC_VERSION@.so
+   obj:/*libc-@GLIBC_VERSION@*.so
 }
 
 
@@ -199,18 +199,18 @@
    Memcheck:Param
    socketcall.sendto(msg)
    fun:send
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
 }
 {
    X11-64bit-padding-4b
    Memcheck:Param
    socketcall.send(msg)
    fun:send
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
-   obj:/*libc-@GLIBC_VERSION@.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
+   obj:/*libc-@GLIBC_VERSION@*.so
 }
 
 ##----------------------------------------------------------------------##
